packages/core/src/external/ehr/unit-conversion.ts

```diff
@@ -204,3 +204,7 @@ function isBpm(units: string): boolean {
 function isPercent(units: string): boolean {
   return units === "%";
 }
+
+export function convertNumberToIntegerOrDecimalString(value: number): string {
```

Comment:
can we call this formatNumberSimple? Think its easier to click into the function and see what its doing that try to read and guess at the long name. 

My pref is to give small names if inner code is trivial as people can always click in and check it.